plugins/typing-break/msd-typing-break-manager.c

```diff
@@ -222,22 +222,14 @@ msd_typing_break_manager_class_init (MsdTypingBreakManagerClass *klass)
 static void
 msd_typing_break_manager_init (MsdTypingBreakManager *manager)
 {
-        manager->priv = msd_typing_break_manager_get_instance_private (manager);
-
 }
```

Comment:
Build warning https://travis-ci.org/github/mate-desktop/mate-settings-daemon/jobs/755671625#L5193
```
msd-typing-break-manager.c: In function 'msd_typing_break_manager_init':

msd-typing-break-manager.c:223:55: warning: unused parameter 'manager' [-Wunused-parameter]

  223 | msd_typing_break_manager_init (MsdTypingBreakManager *manager)

      |                                ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~
```
Now we have a empty function, not sure if this can be fixed.